src/assets/pages/FightingArenaPage/FightingArenaPage.tsx

```diff
@@ -174,61 +214,57 @@ function FightingArenaPage() {
         />
       </PokemonDropdownContainer>
 
-      {fight && <FightArea>
-        {fight.status === FightStatus.WON || fight.status === FightStatus.LOST || fight.status === FightStatus.FLED ? (
-          <WinLossComponent status={fight.status} onReset={handleReset} />
-        ) : (<>
-          <GamingArea>
-            <GamingCard
-              header="You"
-              pokemon={selectedPokemon}
-            />
-            {fight.status === FightStatus.PRE_FIGHT ? (
-              <FightButton
-                variant={ButtonVariant.PRIMARY}
-                size={ButtonSize.SMALL}
-                onClick={handleFightButtonClick}
-              >
-                {FIGHT_BUTTON_TEXT}
-              </FightButton>
-            ) : (
-              <InFightButtonsWrapper>
-                <InFightButton
-                  variant={ButtonVariant.PRIMARY}
-                  size={ButtonSize.SMALL}
-                  onClick={handleAttackClick}
-                  isDisabled={fight.status !== FightStatus.IN_FIGHT}
-                >
-                  {ATTACK_BUTTON_TEXT}
-                </InFightButton>
-                <InFightButton
-                  variant={ButtonVariant.PRIMARY}
-                  size={ButtonSize.SMALL}
-                  onClick={handleCatchClick}
-                  isDisabled={fight.status !== FightStatus.IN_FIGHT}
-                  disabled={fight.status !== FightStatus.IN_FIGHT}
-                  isSucess={fight.status === FightStatus.CATCHED}
-                >
-                  {CATCH_BUTTON_TEXT}
-                </InFightButton>
-              </InFightButtonsWrapper>
-            )}
-            <GamingCard
-              header="Oppeonent"
-              pokemon={opponentPokemon}
-            />
-          </GamingArea>
-         { fight.status !== FightStatus.PRE_FIGHT &&
-          <GameStatusDisplay text={gameStatusText}/>}
-          </>
-        )}
-      </FightArea>}
-      
+      {fight && (
+        <FightArea>
+          {fight.status === FightStatus.WON ||
+          fight.status === FightStatus.LOST ||
+          fight.status === FightStatus.FLED ? (
+            <WinLossComponent status={fight.status} onReset={handleReset} />
```

Comment:
Create an array of fightEndedStatuses
```
const fightEndedStatuses = [FightStatus.WON, FightStatus.LOST, FightStatus.FLED];
```

and check if the array includes fight.status


```suggestion
{fight && (
  <FightArea>
    {fightEndedStatuses.includes(fight.status) ? (
      <WinLossComponent status={fight.status} onReset={handleReset} />
```